cl/recap/tasks.py

```diff
@@ -1747,6 +1754,20 @@ def mark_fq_status(fq, msg, status):
         fq.date_completed = now()
     fq.save()
 
+    # Send webhook event when the fetch task is completed, only send it for
+    # successful or failed like statuses.
+    if fq.status in [
+        PROCESSING_STATUS.SUCCESSFUL,
+        PROCESSING_STATUS.FAILED,
+        PROCESSING_STATUS.INVALID_CONTENT,
+        PROCESSING_STATUS.NEEDS_INFO,
+    ]:
+        user_webhooks = fq.user.webhooks.filter(
+            event_type=WebhookEventType.RECAP_FETCH, enabled=True
+        )
+        for user_webhook in user_webhooks:
+            send_recap_fetch_webhook_event(user_webhook, fq)
+
```

Comment:
Done, I've created the method: `recap_fetch_webhook_check` maybe it would be good to move this method also within `cl.api.webhooks`. If that's ok I'll move it once #2456 is merged here, to avoid merge conflicts.